android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -238,4 +243,54 @@ public void destroy()
             adView = null;
         }
     }
+
+    @Override
+    public void onAdLoaded(final MaxAd ad)
```

Comment:
In our codebase we usually order callbacks in the order which they occur:

[loaded \ failed to load] -> [displayed / display failed] -> [clicked] -> [expand] -> [contract] -> [hidden]

the "ad displayed" and "ad hidden" callbacks can be ignored for BMLs tough: 

https://dash.applovin.com/documentation/mediation/android/ad-formats/banners

Reply:
Updated the orders.